src/Evolution/DiscontinuousGalerkin/MortarData.cpp

```diff
@@ -80,6 +80,11 @@ void MortarData<Dim>::insert_local_geometric_quantities(
                 "passed in for the face Jacobian determinant and normal vector "
                 "magnitude ("
              << local_face_det_jacobian[0].size() << ")");
+  ASSERT(not using_only_face_normal_magnitude_,
+         "The face normal, volume inverse Jacobian determinant, and face "
+         "Jabocian determinant  cannot be inserted because the only the face "
```

Comment:
remove the extra space between "determinant  cannot"

Reply:
ping on this when squashing

Reply:
:+1: I also changed `Jabocian` to `Jacobian` :banana: